### PR TITLE
Todo Complete project:

### PR DESCRIPTION
**Pull request by MrOmachi**
### In this PR, I have:

- Used webpack to bundle JavaScript.

- Learned how to use proper ES6 syntax.
- Used ES6 modules to write modular JavaScript.
- Added some functionality to my application to make it interactive
- Updated items object's value for completed key upon user actions.
- Implemented a function for the "Clear all completed" button

### CODE REVIEW

File: src/modules/localStorage.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          localStorage.setItem('todoL', JSON.stringify(newArr));
+        }
+        return newArr;
+      });
+      window.location.reload();
+    } else {
+      localStorage.removeItem('todoL');
+      window.location.reload();
+    }
+  }
```
Comment (Reem-lab) [inline src/modules/localStorage.js]: - [x] I think the id/indexes is not updated as writen in the requirements  so when deleteing atask it should take the right index to be consecutive numbers from (1-n)
image


https://user-images.githubusercontent.com/58553711/184270732-7e7f99fe-8b88-4133-98ca-16a66e15462d.mp4

File: src/modules/localStorage.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  static remove(id) {
+    const todoL = Storage.getToDo();
+    id = Number(id);
+    todoL.forEach((todo, i) => {
+      if (todo.id === id) {
+        todoL.splice(i, 1);
+      }
+    });
+    localStorage.setItem('todoL', JSON.stringify(todoL));
+  }
```
Comment (Reem-lab) [inline src/modules/localStorage.js]: - [ ] Great work on implementing the functionality to remove one task but I can't see any button or any icon to remove a specific task so kindly add it so I can be able to see if the functionality works fine. 👌🏻🙏🏻✅

- [ ] When I press on the checkbox it converts the completed to true that's right but if I unchecked on the checkbox it doesn't return back, so kindly work on the completed functionality as well 🙏🏻💯

image

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @MrOmachi, 👋🏻👋🏻 

Good job so far!🔥🔥🚀🚀
There are some issues that you still need to work on to go to the next project but you are almost there!


Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality ✅
📌 Good commit messages ✅
📌 Professional README file. ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/localStorage.js (hunk 1 above)
Comment (MrOmachi) [reply src/modules/localStorage.js]: Resolved, items marked completed delete from storage one at a time

File: src/modules/localStorage.js (hunk 2 above)
Comment (MrOmachi) [reply src/modules/localStorage.js]: create an x delete button to remove itms one at a time from storage

**Review by EricMbouwe — APPROVED** [review]
Hi @MrOmachi, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**